lib/data/benefit_cap_data.yml

```diff
@@ -1,4 +1,21 @@
 ---
-single: 350
-couple: 500
-parent: 500
+rates:
```

Comment:
As suggested in the original commit, I'd name this `weekly_benefit_cap` or similar.